libbeat/processors/script/javascript/module/console/console.go

```diff
@@ -0,0 +1,84 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package console
+
+import (
+	"github.com/dop251/goja"
+	"github.com/dop251/goja_nodejs/require"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/pkg/errors"
+
+	// Require the util module for handling the log format arguments.
+	_ "github.com/dop251/goja_nodejs/util"
+)
+
+// Console is a module that enables logging via the logp package (Beat logger).
+type Console struct {
+	runtime *goja.Runtime
+	util    *goja.Object
+}
+
+func (c *Console) makeLogFunc(level logp.Level) func(call goja.FunctionCall) goja.Value {
+	return func(call goja.FunctionCall) goja.Value {
+		if format, ok := goja.AssertFunction(c.util.Get("format")); ok {
+			ret, err := format(c.util, call.Arguments...)
+			if err != nil {
+				panic(err)
+			}
+
+			log := logp.L().Named("processor.javascript")
+			switch level {
+			case logp.DebugLevel:
```

Comment:
Are we missing `Info`?

Reply:
I'm glad you brought this up. For some reason I had it in my head that the typical console API in the browser didn't have an `info`. But I was wrong. https://developer.mozilla.org/en-US/docs/Web/API/Console/info

I'll add this.